tests/framework/installer/ceph_installer.go

```diff
@@ -652,6 +671,12 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(manifests ...CephManifests)
 
 		// helm cleanup
 		if h.settings.UseHelm {
+			if h.settings.RetainHelmDefaultStorageCRs {
```

Comment:
You already deleted these resources above on line 637, right? No need to delete again here.

Reply:
yes your right, my overzealous ctrl-v

Reply:
sorry for using up all your CI time!

Reply:
ha, it's for a good cause!